CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
   [Max Langer](https://github.com/mangerlahn)
   [#853](https://github.com/CocoaPods/Xcodeproj/pull/853) 
 
+* file reference: Remove build files when underlying file_ref is removed.
```

Comment:
nit: please add two empty spaces at the end of this line for markdown formatting to work properly.